tasks/internal_reconcile_report_mismatch/schemas/output.json

```diff
@@ -0,0 +1,95 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "$id": "https://github.com/nasa/cumulus-orca/blob/master/tasks/internal_reconcile_report_mismatch/schemas/output.json",
+    "title": "internal_reconcile_report_mismatch Lambda Output",
+    "description": "The output for the internal_reconcile_report_mismatch Lambda.",
+    "type": "object",
+    "required": [
+      "jobId",
+      "anotherPage",
+      "mismatches"
+    ],
+    "properties": {
+        "jobId": {
+            "type": "integer",
+            "description": "The unique ID of the reconciliation job."
+          },
+        "anotherPage": {
+          "description": "Indicates if more results can be retrieved on another page.",
+          "type": "boolean"
+        },
+        "mismatches": {
+          "type": "array",
+          "items": {
```

Comment:
Should discuss this output at our architecture meeting and possibly working group, among items to think about
- Paging information (should be more robust)
- Mismatch information (wondering if we should limit this a bit or use GraphQL) Think about what we would display on a webpage for an end user.
- Some of these items may be null so I am not sure all the fields you specified would be required.

Reply:
I hesitate to add research/development/standards deviations in a project that's already behind schedule.

Reply:
based on the mismatch table, all those fields are given as not Null

https://github.com/nasa/cumulus-orca/blob/develop/website/static/img/Research-Reconciliation-orca-internal-reconciliation-schema.png

Reply:
need to discuss with working group/operators on what should be returned.

Reply:
Created https://bugs.earthdata.nasa.gov/browse/ORCA-412
https://bugs.earthdata.nasa.gov/browse/ORCA-413